iocage/lib/Jail.py

```diff
@@ -673,9 +673,15 @@ def devfs_ruleset(self) -> iocage.lib.DevfsRules.DevfsRuleset:
         devfs_ruleset = iocage.lib.DevfsRules.DevfsRuleset()
         devfs_ruleset.clone(configured_devfs_ruleset)
 
-        if self._dhcp_enabled:
+        if self._dhcp_enabled is True:
             devfs_ruleset.append("add path 'bpf*' unhide")
 
+        if self.config["allow_usb"] is True:
+            devfs_ruleset.append("add path 'usb/*' unhide")
```

Comment:
Need to investigate the security impact by unhiding `/dev/usb/*`

Reply:
why is this even necessary after the lines below?